src/dynamics/ads_player/ads_player.js

```diff
@@ -331,22 +338,33 @@ Scoped.define("module:Ads.Dynamics.Player", [
 
                 create: function() {
                     const dynamics = this.parent();
-                    const isMobile = Info.isMobile();
                     const adContainer = this.getAdContainer();
+                    const IMASettings = this.get("imasettings");
                     const adManagerOptions = {
                         adContainer: adContainer,
                         adsRenderingSettings: this.get("imaadsrenderingsetting"),
-                        IMASettings: this.get("imasettings")
-                    };
-
-                    if (isMobile && Info.iOSversion().major >= 10) {
-                        adManagerOptions.IMASettings.iOS10Plus = true;
+                        videoElement: this.getVideoElement() || null,
+                        IMASettings
                     }
 
-                    if (!isMobile && this.getVideoElement()) {
-                        // It's optionalParameter
-                        adManagerOptions.videoElement = this.getVideoElement();
+                    if (Info.isMobile()) {
```

Comment:
Really want to abstract this out but not sure if it is worth doing and I could not think of a good name for the method... Maybe something like `normalizeOptionsForMobile`??

Reply:
That's a descriptive name and works for me!